zerver/webhooks/jsonprinter/view.py

```diff
@@ -0,0 +1,39 @@
+import json
+from typing import Any, Dict
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import REQ, has_request_variables, webhook_view
+from zerver.lib.response import json_success
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.models import UserProfile
+
+JSONPRINTER_MESSAGE_TEMPLATE = """
+```
+{webhook_payload}
+```
+""".strip()
+
+
+@webhook_view("JsonPrinter")
+@has_request_variables
+def api_jsonprinter_webhook(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    payload: Dict[str, Any] = REQ(argument_type="body"),
+) -> HttpResponse:
+
+    body = get_body_for_http_request(payload)
+    subject = get_subject_for_http_request(payload)
+
+    check_send_webhook_message(request, user_profile, subject, body)
+    return json_success()
+
+
+def get_subject_for_http_request(payload: Dict[str, Any]) -> str:
+    return "JSON Printer"
```

Comment:
You can probably hardcode this rather than having a function.

Reply:
The only reason I wanted to keep in the function, is so that in future if it needs changing at all, it's in the standard that we use for the rest. Thought it makes it more consistent. Happy to change if you disagree.

Reply:
Yeah, that's fine.